diff -c -r librat/bpms/src/lib/alloc/makefile librat-mingw32/bpms/src/lib/alloc/makefile
*** librat/bpms/src/lib/alloc/makefile	2009-09-11 11:02:28.000000000 +0200
--- librat-mingw32/bpms/src/lib/alloc/makefile	2012-01-07 23:01:13.199413358 +0100
***************
*** 22,30 ****
  liballoc:		makefile ${OBJ}/allocate.o
  		@echo ""
  		@echo "archiving memory allocation library"
! 		ar rcv ${LIBRARY}/liballoc_${ARCH}.a ${OBJ}/allocate.o 
! 		ranlib ${LIBRARY}/liballoc_${ARCH}.a
! 		$(DYN) -o ${LIBRARY}/liballoc_${ARCH}.so ${OBJ}/allocate.o
  
  # object building
  
--- 22,30 ----
  liballoc:		makefile ${OBJ}/allocate.o
  		@echo ""
  		@echo "archiving memory allocation library"
! 		$(AR) rcv ${LIBRARY}/liballoc_${ARCH}.a ${OBJ}/allocate.o 
! 		$(RANLIB) ${LIBRARY}/liballoc_${ARCH}.a
! #		$(DYN) -o ${LIBRARY}/liballoc_${ARCH}.so ${OBJ}/allocate.o
  
  # object building
  
