*** librat/bpms/src/start/makefile	2010-04-09 12:04:23.000000000 +0200
--- librat-mingw32/bpms/src/start/makefile	2012-01-11 09:03:49.767319486 +0100
***************
*** 10,28 ****
  BIN = ../../bin
  LEWIS_LIB = "${BPMS}/src"
  
! LIBS = -limage_${ARCH} -lmatrix_${ARCH} -lalloc_${ARCH} -lvect_${ARCH} -lerr_${ARCH} -lhipl_${ARCH} -lrand_${ARCH} -lrat_${ARCH}  -lm -lc 
  LIBRARY = ${LEWIS_LIB}
  CFLAGS += -I../lib/rat -L${LEWIS_LIB}/lib -DDOUBLEDEF  -DMATCHK -D${ARCH} -D_NO_NAG
  
  #CFLAGS += -O
  
! THIS = start
  
  #	build executables
  
  THIS:		
  		make clean ${THIS} install
  
  
  # conical tree test
  HET01_DIS_ERE:		${THIS}
--- 10,30 ----
  BIN = ../../bin
  LEWIS_LIB = "${BPMS}/src"
  
! LIBS = -limage_${ARCH} -lmatrix_${ARCH} -lalloc_${ARCH} -lvect_${ARCH} -lerr_${ARCH} -lhipl_${ARCH} -lrand_${ARCH} -lrat_${ARCH}  -lm
  LIBRARY = ${LEWIS_LIB}
  CFLAGS += -I../lib/rat -L${LEWIS_LIB}/lib -DDOUBLEDEF  -DMATCHK -D${ARCH} -D_NO_NAG
  
  #CFLAGS += -O
  
! THIS = ratstart
  
  #	build executables
  
  THIS:		
  		make clean ${THIS} install
  
+ ${THIS}.exe: ${THIS}
+ 	mv ${THIS} ${THIS}.exe
  
  # conical tree test
  HET01_DIS_ERE:		${THIS}
***************
*** 37,43 ****
  # object building
  
  .c.o:			makefile $<
! 			$(CC) -I../lib/rat -I/usr/5include -D${ARCH} ${CFLAGS} -I${LIBRARY}/lib -o $@ -c $< 
  
  .f.o:			makefile $<
  			${FORTRAN} ${FFLAGS} -I${LIBRARY}/lib -c $< 
--- 39,45 ----
  # object building
  
  .c.o:			makefile $<
! 			$(CC) -c -I../lib/rat -I/usr/5include -D${ARCH} ${CFLAGS} -I${LIBRARY}/lib -o $@ $< 
  
  .f.o:			makefile $<
  			${FORTRAN} ${FFLAGS} -I${LIBRARY}/lib -c $< 
