diff -c -r librat/bpms/src/lib/r250c/makefile librat-mingw32/bpms/src/lib/r250c/makefile
*** librat/bpms/src/lib/r250c/makefile	2009-09-11 11:03:37.000000000 +0200
--- librat-mingw32/bpms/src/lib/r250c/makefile	2012-01-07 23:00:48.555077311 +0100
***************
*** 22,30 ****
  
  librand:        makefile ${OBJ}/r250.o ${OBJ}/randlcg.o
  				@echo "^Gmaking random interface library"
! 				ar rcv ${LIBRARY}/librand_${ARCH}.a ${OBJ}/r250.o ${OBJ}/randlcg.o
! 				ranlib ${LIBRARY}/librand_${ARCH}.a
! 				$(DYN) -o ${LIBRARY}/librand_${ARCH}.so ${OBJ}/r250.o ${OBJ}/randlcg.o
  
  test:
  				make r250
--- 22,30 ----
  
  librand:        makefile ${OBJ}/r250.o ${OBJ}/randlcg.o
  				@echo "^Gmaking random interface library"
! 				$(AR) rcv ${LIBRARY}/librand_${ARCH}.a ${OBJ}/r250.o ${OBJ}/randlcg.o
! 				$(RANLIB) ${LIBRARY}/librand_${ARCH}.a
! #				$(DYN) -o ${LIBRARY}/librand_${ARCH}.so ${OBJ}/r250.o ${OBJ}/randlcg.o
  
  test:
  				make r250
