diff -c -r librat/bpms/src/lib/rat/imagelib.c librat-mingw32/bpms/src/lib/rat/imagelib.c
*** librat/bpms/src/lib/rat/imagelib.c	2008-02-19 14:39:17.000000000 +0100
--- librat-mingw32/bpms/src/lib/rat/imagelib.c	2012-01-06 23:14:05.522128139 +0100
***************
*** 3,9 ****
--- 3,11 ----
  #include <stdlib.h>
  #include <sys/types.h>
  #include <unistd.h>
+ #ifdef MMAP
  #include <sys/mman.h>
+ #endif
  #include <sys/stat.h>
  #include <fcntl.h>
  #define IMAGEFORMATS_H_BETTER
