diff -c -r librat/bpms/src/lib/vect/makefile librat-mingw32/bpms/src/lib/vect/makefile
*** librat/bpms/src/lib/vect/makefile	2009-09-11 11:04:26.000000000 +0200
--- librat-mingw32/bpms/src/lib/vect/makefile	2012-01-07 23:01:33.081395922 +0100
***************
*** 19,27 ****
  libvect:	makefile ${OBJ}/4D_vectors.o ${OBJ}/vectors2.o ${OBJ}/2D_vectors.o ${OBJ}/2D_vectors_double.o ${OBJ}/3D_vectors_double.o
  		@echo ""
  		@echo "archiving vector/matrix library"
! 		ar rcv ${LIBRARY}/libvect_${ARCH}.a ${OBJ}/2D_vectors_double.o ${OBJ}/4D_vectors.o ${OBJ}/3D_vectors_double.o ${OBJ}/vectors2.o ${OBJ}/2D_vectors.o
! 		ranlib ${LIBRARY}/libvect_${ARCH}.a
! 		$(DYN) -o ${LIBRARY}/libvect_${ARCH}.so ${OBJ}/2D_vectors_double.o ${OBJ}/4D_vectors.o ${OBJ}/3D_vectors_double.o ${OBJ}/vectors2.o ${OBJ}/2D_vectors.o
  
  
  # object building
--- 19,27 ----
  libvect:	makefile ${OBJ}/4D_vectors.o ${OBJ}/vectors2.o ${OBJ}/2D_vectors.o ${OBJ}/2D_vectors_double.o ${OBJ}/3D_vectors_double.o
  		@echo ""
  		@echo "archiving vector/matrix library"
! 		$(AR) rcv ${LIBRARY}/libvect_${ARCH}.a ${OBJ}/2D_vectors_double.o ${OBJ}/4D_vectors.o ${OBJ}/3D_vectors_double.o ${OBJ}/vectors2.o ${OBJ}/2D_vectors.o
! 		$(RANLIB) ${LIBRARY}/libvect_${ARCH}.a
! #		$(DYN) -o ${LIBRARY}/libvect_${ARCH}.so ${OBJ}/2D_vectors_double.o ${OBJ}/4D_vectors.o ${OBJ}/3D_vectors_double.o ${OBJ}/vectors2.o ${OBJ}/2D_vectors.o
  
  
  # object building
