diff -c -r librat/bpms/src/lib/rat/makefile librat-mingw32/bpms/src/lib/rat/makefile
*** librat/bpms/src/lib/rat/makefile	2010-04-09 11:29:55.000000000 +0200
--- librat-mingw32/bpms/src/lib/rat/makefile	2012-01-07 23:01:25.143848085 +0100
***************
*** 25,31 ****
  # make sure program barfs if NaN & doesn't remain tolerant
  #CFLAGS +=  -L.. -DDOUBLEDEF -DMATCHK -D${ARCH} -D_NO_NAG
  #CC = gcc  -Wall -ansi -pedantic
! CFLAGS += -L.. -DDOUBLEDEF -DMMAP  -DMATCHK -D${ARCH} -D_NO_NAG
  
  
  NAG = -lfor -lc
--- 25,31 ----
  # make sure program barfs if NaN & doesn't remain tolerant
  #CFLAGS +=  -L.. -DDOUBLEDEF -DMATCHK -D${ARCH} -D_NO_NAG
  #CC = gcc  -Wall -ansi -pedantic
! CFLAGS += -L.. -DDOUBLEDEF -DMATCHK -D${ARCH} -D_NO_NAG
  
  
  NAG = -lfor -lc
***************
*** 50,60 ****
  
  librat:		makefile ${OBJ}/fratP.o $(USRHIPL) $(PRATOBJ) $(USEFUL) $(WAVEFRONT) ${FORTRANSTUFF} ${OBJ}/matherr.o 
  		make ${OBJ}/start.o
! 		ar rcv ${LIBRARY}/librat_${ARCH}.a ${OBJ}/fratP.o $(USRHIPL) $(PRATOBJ) $(USEFUL) $(WAVEFRONT) ${FORTRANSTUFF} ${OBJ}/matherr.o
! 		ranlib ${LIBRARY}/librat_${ARCH}.a 
! 		ar rcv libstart_${ARCH}.a ${OBJ}/start.o
! 		ranlib libstart_${ARCH}.a
! 		$(DYN) -o ${LIBRARY}/librat_${ARCH}.so ${OBJ}/fratP.o $(USRHIPL) $(PRATOBJ) $(USEFUL) $(WAVEFRONT) ${FORTRANSTUFF} ${OBJ}/matherr.o ${DYN2}
  # object building
  
  #${OBJ}/rpv.o:	rpv.f
--- 50,60 ----
  
  librat:		makefile ${OBJ}/fratP.o $(USRHIPL) $(PRATOBJ) $(USEFUL) $(WAVEFRONT) ${FORTRANSTUFF} ${OBJ}/matherr.o 
  		make ${OBJ}/start.o
! 		$(AR) rcv ${LIBRARY}/librat_${ARCH}.a ${OBJ}/fratP.o $(USRHIPL) $(PRATOBJ) $(USEFUL) $(WAVEFRONT) ${FORTRANSTUFF} ${OBJ}/matherr.o
! 		$(RANLIB) ${LIBRARY}/librat_${ARCH}.a 
! 		$(AR) rcv libstart_${ARCH}.a ${OBJ}/start.o
! 		$(RANLIB) libstart_${ARCH}.a
! #		$(DYN) -o ${LIBRARY}/librat_${ARCH}.so ${OBJ}/fratP.o $(USRHIPL) $(PRATOBJ) $(USEFUL) $(WAVEFRONT) ${FORTRANSTUFF} ${OBJ}/matherr.o ${DYN2}
  # object building
  
  #${OBJ}/rpv.o:	rpv.f
