diff -c -r librat/bpms/src/lib/hips/hipstats.c librat-mingw32/bpms/src/lib/hips/hipstats.c
*** librat/bpms/src/lib/hips/hipstats.c	2009-04-15 18:56:13.000000000 +0200
--- librat-mingw32/bpms/src/lib/hips/hipstats.c	2012-01-07 23:06:12.954234598 +0100
***************
*** 4,12 ****
  #include <math.h>
  #include <limits.h>
  #include "hipl_format.h"
! #ifndef MAXFLOAT 
! #include <values.h>
! #endif
  /* O/P to stdout rather than stderr changed by Lewis July 22 1999 */
  #ifndef _stderr_
  #define OPCHANNEL stdout
--- 4,10 ----
  #include <math.h>
  #include <limits.h>
  #include "hipl_format.h"
! #include <float.h>
  /* O/P to stdout rather than stderr changed by Lewis July 22 1999 */
  #ifndef _stderr_
  #define OPCHANNEL stdout
***************
*** 249,255 ****
  {
    register i=0,j=0,n=0;
    double mean=0., sd=0., tot=0., sqrTot=0., pixVal=0.;
!   double min=MAXFLOAT, max=-MAXFLOAT;
  
    for(i=0;i<h->rows;i++){
      for(j=0;j<h->cols;j++){
--- 247,253 ----
  {
    register i=0,j=0,n=0;
    double mean=0., sd=0., tot=0., sqrTot=0., pixVal=0.;
!   double min=FLT_MAX, max=-FLT_MAX;
  
    for(i=0;i<h->rows;i++){
      for(j=0;j<h->cols;j++){
***************
*** 380,386 ****
  {
    register i=0,j=0,n=0;
    float mean=0., sd=0., tot=0., sqrTot=0., pixVal=0.;
!   float min=MAXFLOAT, max=-MAXFLOAT;
    
    
    for(i=0;i<h->rows;i++){
--- 378,384 ----
  {
    register i=0,j=0,n=0;
    float mean=0., sd=0., tot=0., sqrTot=0., pixVal=0.;
!   float min=FLT_MAX, max=-FLT_MAX;
    
    
    for(i=0;i<h->rows;i++){
***************
*** 424,430 ****
    register i=0,j=0,l=0,n=0;
    int nLines=0;
    double mean=0., sd=0., tot=0., sqrTot=0., pixVal=0.;
!   double min=MAXFLOAT, max=-MAXFLOAT;
    unsigned char *line=NULL, *liney=NULL;
    static int num=0;
  
--- 422,428 ----
    register i=0,j=0,l=0,n=0;
    int nLines=0;
    double mean=0., sd=0., tot=0., sqrTot=0., pixVal=0.;
!   double min=FLT_MAX, max=-FLT_MAX;
    unsigned char *line=NULL, *liney=NULL;
    static int num=0;
  
