diff -c -r librat/bpms/src/lib/rat/start.c librat-mingw32/bpms/src/lib/rat/start.c
*** librat/bpms/src/lib/rat/start.c	2011-07-19 16:20:50.000000000 +0200
--- librat-mingw32/bpms/src/lib/rat/start.c	2012-01-07 23:43:15.313726203 +0100
***************
*** 517,530 ****
--- 517,533 ----
  
  void userSignals(){
  	/* always call RATsignalInterrupt */
+ #ifndef WIN32
  	signal(SIGUSR1,RATsignalInterrupt);
  	signal(SIGUSR2,RATsignalInterrupt);
  	signal(SIGSTOP,RATsignalInterrupt);
+ #endif
          signal(SIGSEGV,RATsignalInterrupt);
  }
  
  void RATuserInterrupt(RATobj *ratObj,int sig){
  	switch(sig){
+ #ifndef WIN32
  		case SIGUSR1:
  			fprintf(stderr,"rat:\tsignal interrupt %d for process %d\n",sig,RATgetPID());
  			break;
***************
*** 535,540 ****
--- 538,544 ----
  			fprintf(stderr,"rat:\tsignal interrupt %d for process %d\n",sig,RATgetPID());
  			exit(1);
  			break;
+ #endif
  		case SIGSEGV:
  			fprintf(stderr,"rat:\tsignal interrupt %d for process %d\n(try switching it off and on again ;-) )\n",sig,RATgetPID());
  			exit(1);
