--- librat/bpms/src/lib/rat/envilib.c.orig	2013-10-01 10:13:08.210728672 +0200
+++ librat-mingw32/bpms/src/lib/rat/envilib.c	2013-10-01 10:11:07.037784158 +0200
@@ -7,6 +7,46 @@
 #include "hiplib.h"
 
 
+#ifdef __MINGW32__
+/* This code is public domain -- Will Hartung 4/9/09 */
+size_t getline(char **lineptr, size_t *n, FILE *stream) {
+  char *bufptr = NULL; 
+  char *p = bufptr;
+  size_t size;
+  int c;
+
+  if (lineptr == NULL) { return -1; } 
+  if (stream == NULL)  { return -1; }
+  if (n == NULL)       { return -1; }
+  bufptr = *lineptr;
+  size = *n;
+  c = fgetc(stream);
+  if (c == EOF)        { return -1; }
+  if (bufptr == NULL) { 
+    bufptr = malloc(128);
+    if (bufptr == NULL) { return -1; }
+    size = 128;
+  } 
+  p = bufptr;
+  while(c != EOF) { 
+    if ((p - bufptr) > (size - 1)) {
+      size = size + 128;
+      bufptr = realloc(bufptr, size);
+      if (bufptr == NULL) { return -1; }
+    }
+    *p++ = c;
+    if (c == '\n') { break; }
+    c = fgetc(stream);
+  } 
+
+  *p++ = '\0';
+  *lineptr = bufptr;
+  *n = size;
+
+  return p - bufptr - 1;
+}
+#endif
+
 char *getField(char *field,char *data){
   /* assume data contains all of the text we might need
   ** and we want to return a string array of the fields
