--- librat/bpms/src/lib/vect/makefile.orig	2009-09-11 11:04:26.000000000 +0200
+++ librat/bpms/src/lib/vect/makefile	2014-12-17 16:57:26.653652786 +0100
@@ -19,9 +19,9 @@
 libvect:	makefile ${OBJ}/4D_vectors.o ${OBJ}/vectors2.o ${OBJ}/2D_vectors.o ${OBJ}/2D_vectors_double.o ${OBJ}/3D_vectors_double.o
 		@echo ""
 		@echo "archiving vector/matrix library"
-		ar rcv ${LIBRARY}/libvect_${ARCH}.a ${OBJ}/2D_vectors_double.o ${OBJ}/4D_vectors.o ${OBJ}/3D_vectors_double.o ${OBJ}/vectors2.o ${OBJ}/2D_vectors.o
-		ranlib ${LIBRARY}/libvect_${ARCH}.a
-		$(DYN) -o ${LIBRARY}/libvect_${ARCH}.so ${OBJ}/2D_vectors_double.o ${OBJ}/4D_vectors.o ${OBJ}/3D_vectors_double.o ${OBJ}/vectors2.o ${OBJ}/2D_vectors.o
+		$(AR) rcv ${LIBRARY}/libvect_${ARCH}.a ${OBJ}/2D_vectors_double.o ${OBJ}/4D_vectors.o ${OBJ}/3D_vectors_double.o ${OBJ}/vectors2.o ${OBJ}/2D_vectors.o
+		$(RANLIB) ${LIBRARY}/libvect_${ARCH}.a
+#		$(DYN) -o ${LIBRARY}/libvect_${ARCH}.so ${OBJ}/2D_vectors_double.o ${OBJ}/4D_vectors.o ${OBJ}/3D_vectors_double.o ${OBJ}/vectors2.o ${OBJ}/2D_vectors.o
 
 
 # object building
