diff -c -r librat/bpms/src/lib/hips/bs.c librat-mingw32/bpms/src/lib/hips/bs.c
*** librat/bpms/src/lib/hips/bs.c	2006-09-22 15:36:56.000000000 +0200
--- librat-mingw32/bpms/src/lib/hips/bs.c	2012-01-07 23:24:02.096515434 +0100
***************
*** 69,75 ****
  
  	/* read first four bytes (Fortran header) from file, and discard */
   
! 	if(read((float *)in_buf,sizeof(float),rows*cols,in_fd)!=rows*cols){
  		fprintf(stderr,"%s: Error during read of file %s\n",argv[0],in_file);
  		exit(1);
  	}
--- 69,76 ----
  
  	/* read first four bytes (Fortran header) from file, and discard */
   
! 	//if(read((float *)in_buf,sizeof(float),rows*cols,in_fd)!=rows*cols){
! 	if(read(in_fd, (float *)in_buf, sizeof(float)*rows*cols)!=sizeof(float)*rows*cols){
  		fprintf(stderr,"%s: Error during read of file %s\n",argv[0],in_file);
  		exit(1);
  	}
***************
*** 88,94 ****
  
  
  	/* write out data */
! 	if(write(in_buf,sizeof(float),rows*cols,out_fd)!=rows*cols){
  		fprintf(stderr,"%s: Error during write of file %s\n",argv[0],out_file);
  		exit(1);	
  	}
--- 89,96 ----
  
  
  	/* write out data */
! 	// if(write(in_buf,sizeof(float),rows*cols,out_fd)!=rows*cols){
! 	if(write(out_fd, in_buf, sizeof(float)*rows*cols)!=sizeof(float)*rows*cols){
  		fprintf(stderr,"%s: Error during write of file %s\n",argv[0],out_file);
  		exit(1);	
  	}
