diff -c -r librat/bpms/src/lib/hips/makefile librat-mingw32/bpms/src/lib/hips/makefile
*** librat/bpms/src/lib/hips/makefile	2009-09-11 11:03:21.000000000 +0200
--- librat-mingw32/bpms/src/lib/hips/makefile	2012-01-10 10:51:11.707062390 +0100
***************
*** 18,26 ****
  		${MAKE} libhipl reseq hipstats linear stripheader bs hips2pbm install
  
  libhipl:	${OBJ}/libhipl.o
! 		ar rcv ${LIBRARY}/libhipl_${ARCH}.a ${OBJ}/libhipl.o
! 		ranlib ${LIBRARY}/libhipl_${ARCH}.a
! 		$(DYN) -o ${LIBRARY}/libhipl_${ARCH}.so ${OBJ}/libhipl.o
  
  install:	reseq hipstats linear stripheader bs hips2pbm
  		mv ${OBJ}/reseq ${OBJ}/hipstats ${OBJ}/linear ${OBJ}/stripheader ${OBJ}/bs ${OBJ}/hips2pbm ../../../bin/${ARCH}
--- 18,26 ----
  		${MAKE} libhipl reseq hipstats linear stripheader bs hips2pbm install
  
  libhipl:	${OBJ}/libhipl.o
! 		$(AR) rcv ${LIBRARY}/libhipl_${ARCH}.a ${OBJ}/libhipl.o
! 		$(RANLIB) ${LIBRARY}/libhipl_${ARCH}.a
! #		$(DYN) -o ${LIBRARY}/libhipl_${ARCH}.so ${OBJ}/libhipl.o
  
  install:	reseq hipstats linear stripheader bs hips2pbm
  		mv ${OBJ}/reseq ${OBJ}/hipstats ${OBJ}/linear ${OBJ}/stripheader ${OBJ}/bs ${OBJ}/hips2pbm ../../../bin/${ARCH}
