diff -c -r librat/bpms/src/lib/image/makefile librat-mingw32/bpms/src/lib/image/makefile
*** librat/bpms/src/lib/image/makefile	2013-03-11 15:30:14.000000000 +0100
--- librat-mingw32/bpms/src/lib/imagemakefile	2013-09-24 16:50:07.691376272 +0200
***************
*** 5,17 ****
  LIBRARY = ${BPMS}/src/lib
  DYN2 = 
  
! CFLAGS += -I. -DMMAP  -DVERBOSE -DDOUBLEDEF -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64
  
  all:	${OBJ}/filelib.o ${OBJ}/hiplib.o ${OBJ}/envilib.o ${OBJ}/imagelib.o
  	mkdir -p ${OBJ}
! 	ar rcv ${LIBRARY}/libimage_${ARCH}.a ${OBJ}/filelib.o ${OBJ}/hiplib.o ${OBJ}/envilib.o ${OBJ}/imagelib.o
! 	$(DYN) -o ${LIBRARY}/libimage_${ARCH}.so ${OBJ}/filelib.o ${OBJ}/hiplib.o ${OBJ}/envilib.o ${OBJ}/imagelib.o ${DYN2}
! 	ranlib ${LIBRARY}/libimage_${ARCH}.a
  
  ${OBJ}/%.o:	%.c
  		mkdir -p ${OBJ}
--- 5,17 ----
  LIBRARY = ${BPMS}/src/lib
  DYN2 = 
  
! CFLAGS += -I. -DVERBOSE -DDOUBLEDEF -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64
  
  all:	${OBJ}/filelib.o ${OBJ}/hiplib.o ${OBJ}/envilib.o ${OBJ}/imagelib.o
  	mkdir -p ${OBJ}
! 	$(AR) rcv ${LIBRARY}/libimage_${ARCH}.a ${OBJ}/filelib.o ${OBJ}/hiplib.o ${OBJ}/envilib.o ${OBJ}/imagelib.o
! #	$(DYN) -o ${LIBRARY}/libimage_${ARCH}.so ${OBJ}/filelib.o ${OBJ}/hiplib.o ${OBJ}/envilib.o ${OBJ}/imagelib.o ${DYN2}
! 	$(RANLIB) ${LIBRARY}/libimage_${ARCH}.a
  
  ${OBJ}/%.o:	%.c
  		mkdir -p ${OBJ}
