--- src/ancillary.c.orig	2012-08-27 11:21:31.000000000 +0200
+++ src/ancillary.c	2013-01-23 11:14:26.301641798 +0100
@@ -50,6 +50,21 @@
 #define s2day   3600.0*24.0 /* seconds to day */
 #define mW2W    1.E-3;       /* mW to W */
 
+#ifdef WIN32
+int setenv(char *name, char *value, int overwrite) {
+  int len = strlen(name)+1+strlen(value)+1;
+  /* apparently leaking memory is required */
+  char *str = malloc(len);
+  sprintf(str, "%s=%s",name,value);
+  putenv(str);
+  return 0;
+}
+int unsetenv(char *name) {
+  int retval = setenv(name, "", 1);
+  return retval;
+}
+#endif
+
 /* prototypes of internal functions */
 static int cnvlv (double *x_spec, float  *y_spec, int n_spec,
 		  double *x_slit, double *y_slit, int n_slit, int std);
